Makefile

```diff
@@ -242,7 +246,7 @@ clean:
 	@$(MAKE) multi-distclean
 	@rm -rf _build/system_test+test _build/system_test system_test/aest_hard_fork_SUITE_data/db system_test/logs _build/test _build/prod _build/local
 	@rm -rf _build/default/plugins
-	@rm -rf $$(ls -d _build/default/lib/* | grep -v '[^_]rocksdb') ## Dependency `rocksdb` takes long to build.
+	@rm -rf $$(ls -d _build/default/lib/* | grep -Ev "$$(sep=""; for d in $(CLEAN_DEPS_EXCEPTIONS); do echo -n "$$sep[^_]$$d"; sep="|"; done)")
```

Comment:
I did not have a chance of trying this locally - in the middle of other task - though this looks ok and POSIX compliant.

Reply:
Why all of this magic instead of `rocksdb|sha3|enacl|erlexec` (kind of) ?